src/features/Students/data/slice.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { RequestStatus } from 'features/constants';
+
+const initialState = {
+  table: {
+    currentPage: 1,
+    data: [],
+    status: RequestStatus.LOADING,
+    error: null,
+    numPages: 0,
+    count: 0,
+  },
+  courses: {
+    data: [],
+    status: RequestStatus.LOADING,
+    error: null,
+  },
+  classes: {
+    data: [],
+    status: RequestStatus.LOADING,
+    error: null,
+  },
+  filters: {},
```

Comment:
Could you please move this to the end of the inititalState. @AuraAlba

Reply:
done